presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -0,0 +1,146 @@
+===============================
+Presto C++ Properties Reference
+===============================
+
+This section describes Presto C++ configuration properties.
+
+The following is not a complete list of all configuration and
+session properties, and does not include any connector-specific
+catalog configuration properties. For information on catalog 
+configuration properties, see :doc:`Connectors </connector/>`.
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Coordinator Properties
+----------------------
+
+Set the following configuration properties for the Presto coordinator exactly 
+as they are shown in this code block to enable the Presto coordinator's use of 
+Presto C++ workers. 
+
+.. code-block:: none
+
+    native-execution-enabled=true
+    optimizer.optimize-hash-generation=false
+    regex-library=RE2J
+    use-alternative-function-signatures=true
+    experimental.table-writer-merge-operator-enabled=false
```

Comment:
There is also inline-sql-functions=false. We should add this as well.

Reply:
@tdcmeehan : As I research this further, seems like the SqlFunctions are internal developer concepts and not visible to the user.

Do you feel this should be documented as well ?

Reply:
This property applies to both SQL functions created through `CREATE FUNCTION` and to functions defined through the function SPI via e.g. `@SqlInvokedScalarFunction`.  In both cases they are **not** internal developer concepts:

1) Functions defined through `CREATE FUNCTION` are user-defined.
2) Functions annotated as `@SqlInvokedScalarFunction` are part of the SPI.  One can add a new SPI module that adds more SQL functions.  Hence, it is not an internal developer concept either.

Reply:
Based on these comments, I am leaving inline-sql-functions out.